src/main/java/tr/com/srdc/cda2fhir/transform/ResourceTransformerImpl.java

```diff
@@ -1005,7 +1050,7 @@ public EntryResult tClinicalDocument2Bundle(ClinicalDocument cdaClinicalDocument
 						if (performer.getAssignedEntity() != null) {
 							EntityResult entityResult = tPerformer12Practitioner(performer, bundleInfo);
 							if (entityResult.hasPractitioner()) {
-								result.updateFrom(entityResult);
+								result.updateFrom(entityResult); // Right here is where a dup org is created
```

Comment:
ok?

Reply:
Oops. Sorry I was debugging. Let me make sure that isn't still there.